Specs/DateTime/German/DateTimeParser.json

```diff
@@ -288,14 +288,14 @@
     ]
   },
   {
-    "Input": "am 15 um 8:00:24",
+    "Input": "am 15. um 8:00:24",
```

Comment:
Same

Reply:
Do we also want to catch typos or incorrect forms? Without the "." it is incorrect and should not be written that way.